applicationconfig/client/README.md

```diff
@@ -0,0 +1,84 @@
+# App Configuration client library for Java
+Azure App Configuration allows developers to store, retrieve and manage access to application settings all in one place. It is easy to set up and simple to use from any application. It gives developers the ability to modify an application's behavior on demand without having to redeploy the application.
+
+Use the client library for Application Configuration to:
```

Comment:
There should be consistency when using the name of the service. In some places you use `App` and in others `Application`.